module/actor/actor.js

```diff
@@ -3215,6 +3193,109 @@ export class GurpsActor extends Actor {
     return isDamageRoll
   }
 
+  _getTagsFromCheck(chatThing, taggedSettings) {
+    let refTags
+
+    const ref = extractRefFromAttributeOrDefense(chatThing)
+    switch (ref) {
+      case ATTRIBUTE.ST:
+      case ATTRIBUTE.DX:
+      case ATTRIBUTE.IQ:
+      case ATTRIBUTE.HT:
+        refTags = taggedSettings[`all${ref.toUpperCase()}Rolls`].split(',').map(it => it.trim().toLowerCase())
+        refTags = refTags.concat(taggedSettings.allAttributesRolls.split(',').map(it => it.trim().toLowerCase()))
+        break
+
+      case ATTRIBUTE.WILL:
+      case ATTRIBUTE.PER:
+      case ATTRIBUTE.FRIGHT_CHECK:
+      case ATTRIBUTE.VISION:
+      case ATTRIBUTE.HEARING:
+      case ATTRIBUTE.TASTE_SMELL:
+      case ATTRIBUTE.TOUCH:
+      case ATTRIBUTE.CR:
+        refTags = taggedSettings[`all${ref.toUpperCase()}Rolls`].split(',').map(it => it.trim().toLowerCase())
+        break
+
+      case ROLL_TYPE.DODGE:
+        refTags = taggedSettings.allDefenseRolls.split(',').map(it => it.trim().toLowerCase())
+        refTags = refTags.concat(
+          taggedSettings[`all${ref.toUpperCase()}Rolls`].split(',').map(it => it.trim().toLowerCase())
+        )
+        break
+
+      case ROLL_TYPE.PARRY:
+        refTags = taggedSettings.allDefenseRolls.split(',').map(it => it.trim().toLowerCase())
+        refTags = refTags.concat(taggedSettings.allParryRolls.split(',').map(it => it.trim().toLowerCase()))
+        break
+
+      case ROLL_TYPE.BLOCK:
+        refTags = taggedSettings.allDefenseRolls.split(',').map(it => it.trim().toLowerCase())
+        refTags = refTags.concat(taggedSettings.allBlockRolls.split(',').map(it => it.trim().toLowerCase()))
+        break
+
+      default:
+        refTags = []
+    }
+    return refTags
+  }
+
+  _getItemRefFromCheck(chatThing) {
```

Comment:
Could you add a description to this function?

Reply:
Same comment as `_getTagsFromCheck`

Reply:
done